dbms/src/Core/BackgroundSchedulePool.cpp

```diff
@@ -192,6 +190,19 @@ BackgroundSchedulePool::~BackgroundSchedulePool()
 }
 
 
+void BackgroundSchedulePool::start()
+{
+    bool expected = false;
+    if (started.compare_exchange_strong(expected, true))
+        return;
```

Comment:
The pool is started by the server startup code, which runs once, in a single thread, so we don't need this check. Or did I miss something?

Reply:
It's true - it's an artifact from a previous fix attempt.